library/src/main/scala/scala/scalajs/runtime/EnvironmentInfo.scala

```diff
@@ -49,4 +49,20 @@ trait EnvironmentInfo extends js.Object {
   def sourceMapper: js.UndefOr[js.Function1[
     js.Array[JSStackTraceElem], js.Array[JSStackTraceElem]]] = js.native
 
+  /** Semantics configuration.
+   *
+   *  @group envInfo
+   */
+  val semantics: EnvironmentInfo.Semantics = js.native
```

Comment:
Yee, I knew it wasn't ideal, but that was the only place to put it, short of introducing an additional IR node :-(
I guess I'll really have to do that, uh?